src/components/modals/modalWrapper.tsx

```diff
@@ -27,14 +35,18 @@ const ModalWrapper: React.FC<ModalWrapperProps> = ({ children, modalKey }) => {
         />
       </Dialog.Overlay>
       {/* aria-describedby={undefined} is used to prevent the modal from being announced as a dialog */}
-      <Dialog.Content asChild aria-describedby={undefined}>
+      <Dialog.Content
+        asChild
+        aria-describedby={undefined}
+        className="max-w-[95%]"
```

Comment:
I'm assuming this is to make sure there's always margin around the modal, but since this is also the screenwidth (right?) I think it would be more clear to say max-w-[95vw]